core/domain/exp_jobs_one_off.py

```diff
@@ -116,10 +128,12 @@ def reduce(key, values):
         yield (key, values)
 
 
-class MathExpressionInputInteractionOneOffJob(jobs.BaseMapReduceOneOffJobManager):  # pylint: disable=line-too-long
```

Comment:
Is this job not required any more?

Reply:
The old job just checked which interactions have equations as inputs, so this job covers that functionality which is why we can remove the old one.